### PR TITLE
update docs for shell auto completion

### PR DESCRIPTION
**Pull request by alexcb**

### CODE REVIEW

**Review by vladaionescu — APPROVED** [review]
Looks good - can you also update `installation.md`? (somewhere in `docs`)

**Comment by vladaionescu** [comment]
Oh - and also `docs/README.md`

File: release/Earthfile [hunk 1]
```diff
@@ -26,6 +26,8 @@ release-github:
         ../+earth-all/* ./release/
     ARG BODY="No details provided"
     RUN --secret GITHUB_TOKEN=+secrets/GITHUB_TOKEN test -n "$GITHUB_TOKEN"
+    RUN ls -la ./release
+    RUN asdf
```
Comment (vladaionescu) [inline release/Earthfile:30]: Unintended?

**Review by vladaionescu — CHANGES_REQUESTED** [review]

File: release/Earthfile (hunk 1 above)
Comment (alexcb) [reply release/Earthfile:30]: oh yikes! `git commit -a` failed me. will fix in master.

Comment (alexcb) [reply release/Earthfile:30]: fixed in https://github.com/earthly/earthly/commit/1f9ca76b97a37a1524a12dc97ceb5b386e911ce6 sorry about that slip up.

**Pull request merged**